src/utils/hooks/useForm/useForm.js

```diff
@@ -0,0 +1,17 @@
+import { useState } from 'react';
+
+export const useForm = (initialState = {}) => {
```

Comment:
### Optional
You could use `Formik` or `React Hook Form` library to do this, the reason is there are no validation or error messages when I trying to add products and there is no stock